db/models/fund.py

```diff
@@ -32,3 +40,4 @@ class Fund(BaseModel):
     owner_organisation_name = Column("owner_organisation_name", db.String(), nullable=False, unique=False)
     owner_organisation_shortname = Column("owner_organisation_shortname", db.String(), nullable=False, unique=False)
     owner_organisation_logo_uri = Column("owner_organisation_logo_uri", db.Text(), nullable=True, unique=False)
+    funding_type = Column("funding_type", SQLAEnum(FundingType, name="fundingtype"), nullable=True, unique=False)
```

Comment:
I think better to have funding type for all the funds because after this it seems we have a type for the funds better to put it now. so we can have field as not nullable. 

if have different views I'm ok to have this as nullable

Reply:
Thanks & yeh agree with your view @nuwan-samarasinghe . I've initially set it to nullable to allow migrating and setting appropriate values in existing environments as described in the commit here https://github.com/communitiesuk/funding-service-design-fund-store/pull/342/commits/96b9c88f6923ad78fbf185020db0b0d39ce4db5c. 

If we've got a more streamlined method for doing that happy to adjust!

Reply:
Our usual approach is to declare it as not null here, but then in the migration:

1. Create a nullable
2. Backfill existing rows with a value (in this case they are all COMPETITIVE)
3. Set it to not null

Reply:
Done in https://github.com/communitiesuk/funding-service-design-fund-store/pull/342/commits/edb2ae84e3d1d62c57b85b67e28c0f394f52d981